pkg/deploy/util/util.go

```diff
@@ -163,6 +164,23 @@ func MakeDeployment(config *deployapi.DeploymentConfig, codec runtime.Codec) (*a
 	return deployment, nil
 }
 
+// mkdeployment returns a deployment
+// Used for testing
+func mkdeployment(version int) api.ReplicationController {
+	deployment, _ := MakeDeployment(deploytest.OkDeploymentConfig(version), api.Codec)
+	return *deployment
+}
+
+// MkDeploymentList returns a list of deployments
+// Used for testing
+func MkDeploymentList(versions ...int) *api.ReplicationControllerList {
```

Comment:
I had moved those funcs there originally but I was getting cyclic imports. I will move them back just to refresh my memory. Most probably will move them back in the tests